src/Modules/Content/Dnn.PersonaBar.Pages/Pages.Web/src/components/PageDetails/PageStandard/PageStandard.jsx

```diff
@@ -22,67 +24,88 @@ class PageDetails extends Component {
         onChangeField("tags", tags.join(","));
     }
 
-    onChangeUrl(event) {
+    onChangeUrl(value) {
         const {onChangeField} = this.props;
-        let value = event.target.value;
-        if (!value.startsWith("/")) {
-            value = "/" + value;
-        }  
         onChangeField("url", value);
     }
 
     render() {
         const {page, errors} = this.props;
         const tags = page.tags ? page.tags.split(",") : [];
+        const TabParameters = {
+            portalId:  -2,
+            cultureCode: "",
+            isMultiLanguage: false,
+            excludeAdminTabs: false,
+            roles: "",
+            sortOrder: 0
+        };
+
+         let TabParameters_1 = Object.assign(Object.assign({}, TabParameters), { disabledNotSelectable: false });
```

Comment:
this needs to be passed to the PagePicker, PortalTabsParameters={TabParameters_1}